### ISSUE TITLE
Same image - Different output

### ISSUE THREAD

**Issue by groot-1313** [affinelayer/pix2pix-tensorflow#100]
After training the model, I tested it from the latest checkpoint. 
The model predicts different outputs for the same input image!!!

**Comment by groot-1313** [comment]
I switched off the dropout to avoid it.

**Comment by julien2512** [comment]
dropout garanties non specialization of each neuron in a NxN matrix (you do not need to apply it on convolutional networks). That mean your network would be more flexible to change.

I do not know your usecase exactly, but you should keep it while learning.
Eventhough, may be it is possible to keep it for the test.

**Comment by mattchiang-gsp** [comment]
@groot-1313 How exactly did you switch off dropout?
